pkg/controllermanager/apis/config/types.go

```diff
@@ -51,6 +51,8 @@ type ControllerManagerControllerConfiguration struct {
 	CloudProfile *CloudProfileControllerConfiguration
 	// ControllerRegistration defines the configuration of the ControllerRegistration controller.
 	ControllerRegistration *ControllerRegistrationControllerConfiguration
+	// Event defines the configuration of the EventControllerConfiguration controller.
```

Comment:
Can you please cross-check the different API versions and use the same doc strings in both?
```suggestion
	// Event defines the configuration of the Event controller.
```